src/client/components/input/select2.js

```diff
@@ -95,6 +126,11 @@ class Select extends React.Component {
 		select.off('change');
 	}
 
+	/**
+	 * Used by React to render the component.
```

Comment:
Describe the component a bit here - what does it look like, and what does it do? (what Leftmost said - why/where would I use the component?)